### PR TITLE
tests: unit tests for status and list commands

### LINKED ISSUE #70: cli: create tests for existing functionalities and commands
After #3 is done, we should test all the functionalities and commands we already have. The following list should summarize the different components we need to test:

- [x] Utils functions
- [x] Validation functions
- [x] `root` functions and general way to test command output
- [x] `close` command
- [x] `du` command
- [x] `info` command
- [ ] `list` command
- [x] `logs` command
- [x] `open` command
- [ ] `status` command
- [x] `ls` command

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #70 

merge after #79

### CODE REVIEW

File: cmd/list_test.go [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+
+var listServerPath = "/api/workflows"
+
+func TestList(t *testing.T) {
+	successResponse := `{
+		"total": 2,
+		"items": [
+			{
+				"created": "2022-07-28T12:04:37",
+			  	"id": "my_workflow_id",
```
Comment (tiborsimko) [inline cmd/list_test.go:29]: Cosmetics: some of these lines are indented using tabs only, some a combination of tabs and spaces.  E.g. the "created" line and "id" line start on the same column, yet use different indentation. Might be good to harmonise.

File: cmd/list_test.go [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+						"total": 2
+					},
+					"run_finished_at": "2022-07-28T12:13:10",
+					"run_started_at": "2022-07-28T12:04:52"
+			  	},
+			  	"size": {
+					"human_readable": "1 KiB",
+					"raw": 1024
+			  	},
+			  	"status": "finished",
```
Comment (tiborsimko) [inline cmd/list_test.go:48]: Cosmetics: extra trailing white space after "status: finished" line 48. You way want to configure the editor to remove trailing whitespace for all our files.

(The same for line 75 after "status: running".)

File: cmd/status_test.go [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+		"id": "my_workflow_id",
+		"name": "my_workflow.10",
+		"status": "finished",
+		"user": "user",
+		"logs": "logs",
+		"progress": {
+			"current_command": "ls",
+			"current_step_name": "step_name",
+			"failed": {
+				"job_ids": [],
```
Comment (tiborsimko) [inline cmd/status_test.go:37]: Cosmetics: trailing whitespace after several lines here.

trailing

File: cmd/list_test.go (hunk 2 above)
Comment (BrunoRosendo) [reply cmd/list_test.go:48]: The editor usually does it but he didn't because it's inside a string. I'll remove them

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**